### ISSUE TITLE
UI Shell - Hover on icon in Header is displaying 'undefined'

### ISSUE THREAD

**Issue by pulakb** [IBM/carbon-components-angular#588]
## Detailed description
> [Image Link](https://files.slack.com/files-pri/T27SFGS2W-FK94HTLKW/image.png) to highlight the issue 
> Hover on the highlighted icons in Header of UI Shell is displaying 'Undefined'

## Steps to reproduce the issue

1. Visit https://angular.carbondesignsystem.com/?path=/story/ui-shell--header
2. Hover over the extreme right hand side two icons
